File: SDLU_Shooting_Server/VO/DataVO.js
class DataVO
{
<<<<<<< HEAD
    constructor(type, id, payload)
=======
    /**
     * @param {string} type
     * @param {string} payload
     */
    constructor(type, payload)
>>>>>>> 48d54cf1ed8b83871d12f05f2482e88a0fb0f3b1
    {
        this.type = type;
        this.id = id;
        this.payload = payload;
    }
}

exports.DataVO = DataVO;

File: SDLU_Shooting/Assets/001_Scripts/Socket/VO/DataVO.cs
using System;

[Serializable]
public class DataVO
{
    public string type;
    public string payload;
    public int id;

    public DataVO(string type, int id, string payload)
    {
        this.type = type;
        this.id = id;
        this.payload = payload;
    }
}